nautobot/extras/models/customfields.py

```diff
@@ -558,14 +558,13 @@ def to_form_field(
             choices = [(cfc.value, cfc.value) for cfc in self.choices.all()]
             default_choice = self.choices.filter(value=self.default).first()
 
-            if not required or default_choice is None:
-                choices = add_blank_choice(choices)
-
             # Set the initial value to the first available choice (if any)
             if set_initial and default_choice:
                 initial = default_choice.value
 
             if self.type == CustomFieldTypeChoices.TYPE_SELECT:
+                if not required or default_choice is None:
+                    choices = add_blank_choice(choices)
                 field_class = CSVChoiceField if for_csv_import else forms.ChoiceField
```

Comment:
This change feels wrong to me - the form field (as opposed to the filter form field) for both Select and MultiSelect custom fields should have a blank choice, shouldn't it? Can you help me understand why this should be specific to Select fields only?

Reply:
By default multi select fields have blanks

Reply:
Adding it makes blank a choice in the multi select, which was causing error